src/programs/filter.ts

```diff
@@ -43,14 +43,13 @@ class Filter implements CommandHandler<DiscordEvent.MESSAGE> {
 
     if (messageContent.match(/(nigga|nigger)/i)) {
       await message.delete();
-      member
+      await member
         .createDM()
         .then((dm) =>
```

Comment:
```suggestion
                .then(async (dm) =>
          await dm.send(
```
This could still crash the bot if the message failed to send if the user has his DM privacy set to friends only

Reply:
This was corrected to me on the Discord, thank you Michel!